database/sqlite3.ksy

```diff
@@ -33,7 +33,7 @@ doc: |
 doc-ref: https://www.sqlite.org/fileformat.html
 seq:
   - id: magic
-    contents: ["SQLite format 3", 0]
+    contents: [SQLite format 3, 0]
```

Comment:
IMHO this change impairs intelligibility - seeing <code>[SQLite format <strong>3, 0</strong>]</code> can lead to false impression that the digit `3` must have the same logical meaning as `0`, because there is nothing that would differentiate them. Sure, if the viewer you use for looking at the code uses advanced YAML highlighting (like VS Code), you'll see a change in color:

![image](https://user-images.githubusercontent.com/47499687/109192495-92b47880-7797-11eb-93c2-15d47c2aafda.png)

But most viewers don't go in so much detail, and some crappy ones (like the Ace editor used in the [Web IDE](https://ide.kaitai.io/devel/)) even do it wrong:

![image](https://user-images.githubusercontent.com/47499687/109192868-0d7d9380-7798-11eb-8fe6-840b7c309da8.png)

So I'd suggest either reverting to the original state:

```suggestion
    contents: ["SQLite format 3", 0]
```

or at least writing `0` in hexadecimal so that it's clear that it's a byte value:

```suggestion
    contents: [SQLite format 3, 0x00]
```